CMSgov/easi-app
# [EASI-2903] Added 'OTHER' trb request type

# EASI-2903

## Changes and Description

- Added `OTHER` TRB Request Type
- Use TRBRequestType objects when generating cards so that they don't later need to be type cast.



## How to test this change

- `scripts/dev up`
- Make a new TRB Request
- Ensure that all the options work (the 2 main cards and the `other` button at the bottom)
- Ensure you can switch request types to/from `OTHER`

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
